tests/integration/test_tcp_handler_interserver_listen_host/test_case.py

```diff
@@ -35,12 +35,23 @@ def start_cluster():
         cluster.shutdown()
 
 
+def requests_get(url, attempts=10, sleep=0.5):
```

Comment:
This will try for 5 seconds and this test is flaky because of those 5 seconds.
You need to increase the range in which we test the request, the retries allow you to finish up earlier if it succeeds.

Reply:
I think it's reasonable time - we don't start trying after server started - we first wait on cluster.start() for TCP on 9000 is started (see below comment from @tavplubix ) and only after that we are trying to connect to whatever we want - 5sec is more than reasonable time I guess

Reply:
>This will try for 5 seconds and this test is flaky because of those 5 seconds.
actually no - it's next test (`test_request_to_node_without_interserver_listen_host`) was flaky - without 5sec - the previous - `test_request_to_node_with_interserver_listen_host` - seems to be fine, but I fixed it anyway

Reply:
Is this helper not enough?

https://github.com/ClickHouse/ClickHouse/blob/79e89cf69cf45e05d51f61e704a7f765e042a423/tests/integration/helpers/cluster.py#L3284-L3312

Reply:
as I remember retries don't work on connection refused error